src/main/scala/io/qbeast/spark/index/writer/SparkDataWriter.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 Qbeast Analytics, S.L.
+ */
+package io.qbeast.spark.index.writer
+
+import io.qbeast.IISeq
+import io.qbeast.core.model.{DataWriter, QTableID, TableChanges}
+import io.qbeast.spark.index.QbeastColumns
+import io.qbeast.spark.index.QbeastColumns.{cubeColumnName, stateColumnName}
+import org.apache.hadoop.mapreduce.Job
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.delta.actions.FileAction
+import org.apache.spark.sql.execution.datasources.OutputWriterFactory
+import org.apache.spark.sql.execution.datasources.parquet.ParquetFileFormat
+import org.apache.spark.sql.functions.col
+import org.apache.spark.sql.types.StructType
+import org.apache.spark.util.SerializableConfiguration
+
+/**
+ * Spark implementation of the DataWriter interface.
+ */
+object SparkDataWriter extends DataWriter[DataFrame, StructType, FileAction] {
+
+  override def write(
```

Comment:
Add scaladocs to the write() method

Reply:
Do override methods need scaladocs? :thinking:

Reply:
You're right, they don't! My bad🙌🏽